group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java

```diff
@@ -505,8 +506,8 @@ public Builder withShareGroupAssignor(ShareGroupPartitionAssignor shareGroupAssi
             return this;
         }
 
-        public Builder withAuthorizer(Authorizer authorizer) {
-            this.authorizer = Optional.of(authorizer);
+        public Builder withAuthorizerPlugin(Plugin<Authorizer> authorizerPlugin) {
+            this.authorizerPlugin = Optional.of(authorizerPlugin);
```

Comment:
We set the parameter is `Optional<Plugin<Authorizer>> authorizerPlugin` in `GroupMetadataManager`, If we consistently pass `Plugin<Authorizer>` into the builder and wrap it inside the method, can we avoid the need to check for null?

Reply:
As far as I can tell the other fields are also always set, so I'll leave the `null` check for consistency.